.github/workflows/verify-tablet-names.yml

```
@@ -0,0 +1,28 @@
+name: Verify Correct Tablet Names
+
+on:
+  push
```

I'm really sorry but I missed the part where you asked if we wanted to have `pull_request` removed as well:

```suggestion
  push:
  pull_request:
```

This should be the final change that is necessary, then we can merge.